docs/proposals/secrets.md

```diff
@@ -0,0 +1,80 @@
+## Motivation
+
+We are provisioning services in Amazon that require highly sensitive credentials to deploy. The way the catalog and broker work currently, the only way to get the credentials to the APBs is by passing them as parameters, which show up in plain text at the service catalog, broker, and APB level. This workflow is also very arduous for users, as they have to copy and paste in boilerplate credentials every time they kick off a deployment.
+
+In addition, a successful deployment of an Amazon Web Service requires intricate knowledge of the environment, existing networking infrastructure resources.  This is information a typical end user will not possess.   We would like this configuration information to be entered once by the cluster administrator and then reused by the broker.
+
+
+#### Examples
+##### Sensitive Information
+- ACCESS_KEY
+- SECRET_KEY
+##### AWS Environment specifics
+- Region
+- VPC_ID
+- Subnets
+- T.B.D. network related variables for RDS, etc.
+
+## Proposed Solution
+
+The Cluster Administrator will create a Secret ahead of time with the data they want to hide from end user.  The Secret will exist in the namespace of the Broker which the end user likely does not have access to.
+The Broker will have a section in it’s configuration file to list Secrets and associate to APBs.  When the Broker runs an APB that has an associated Secret, the Secret is mounted as a file to the running Pod and the APB is run in the Broker’s namespace.
```

Comment:
Why does the broker have a list of secrets? That would mean if you want to add a new secret you have to restart the broker.  Maybe the secret should be in apb.yml?